api/prisma/seed-staging.ts

```diff
@@ -93,6 +93,7 @@ export const stagingSeed = async (
         FeatureFlagEnum.enableCompanyWebsite,
         FeatureFlagEnum.disableCommonApplication,
       ],
+      requiredListingFields: ['name', 'description'],
```

Comment:
nit: "description" isn't a real field for listings, can you change it to "listingsBuildingAddress"